server/src/main/java/io/unitycatalog/server/UnityCatalogServer.java

```diff
@@ -22,24 +22,11 @@
 import io.unitycatalog.server.exception.ErrorCode;
```

Comment:
no tests?

Reply:
The tests are in the next PR, if you'd like to have a single PR, you could directly review the next one i.e. #512

Reply:
I think a better break down of the changes in these 2 PRs would be to add basic server tests in the same PR that modifies server. And additional functionality like CLI with its tests can be the next PR.

Reply:
makes sense, done